values.yaml

```diff
@@ -219,6 +224,22 @@ keda:
       <<: *platformAffinity
     tolerations:
       *platformTolerations
+
+nats:
+  nodeSelector:
+    <<: *platformNodeSelector
```

Comment:
I don't understand the line. Is this a special helm syntax to pull the 'platformNodeSelector' from helper or something?

Reply:
it's copy paste from https://github.com/astronomer/astronomer/blob/8364c76bb7052bb99cb198b87b381aa39199d5c7/values.yaml#L220-L226

it's yaml inheritance syntax http://atechie.net/2009/07/merging-hashes-in-yaml-conf-files/